lib/topic_view.rb

```diff
@@ -623,22 +634,6 @@ def pm_params
     @pm_params ||= TopicQuery.new(@user).get_pm_params(topic)
   end
 
```

Comment:
This can be dropped because PostSerializer is now using `topic_view.post_action_type_view` directly